_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -275,12 +283,53 @@ class HealthCareApp extends React.Component {
           external: this.getExternalData(this.state.applicationData, statePath)
         });
       });
+    }
 
-      if (this.props.location.pathname === '/review-and-submit') {
-        buttons = (<SubmitButton onButtonClick={this.handleSubmit}/>);
-      } else {
-        buttons = (<ContinueButton onButtonClick={this.handleContinue}/>);
-      }
+    // Check which section the user is on and render the correct ProgressButtons.
+    const lastSectionText = (this.getUrl('back')) ? this.getUrl('back').split('/').slice(-1)[0].replace('-', ' ') : '';
+    const nextSectionText = (this.getUrl('next')) ? this.getUrl('next').split('/').slice(-1)[0].replace('-', ' ') : '';
+
+    const backButton = (
+      <ProgressButton
+          onButtonClick={this.handleBack}
+          buttonText={`Back to ${lastSectionText}`}
+          buttonClass={'usa-button-primary-alt'}/>
```

Comment:
I am not crazy about `usa-button-primary-alt` either, especially the weird `active` glow that I see after you click it. I can change this to grey. Where can I find mockups for this project/vets.gov?